client/js/components/Query/StatsBox.js

```diff
@@ -4,6 +4,16 @@ import DistChart from "./StatsBox/DistChart";
 import { Modal } from 'react-bootstrap';
 
 export default class StatsBox extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      buckets: 25
+    };
+  }
+
+  handleChange (e) {
+    this.setState({ buckets: parseInt(e.target.value, 10) });
```

Comment:
The js community has determined that this specific use does not constitute a magic number.